addon-test-support/setup-intl.ts

```diff
@@ -1,5 +1,16 @@
 import addTranslations from './add-translations';
 import { missingMessage } from './-private/serialize-translation';
+import { Translations } from '../addon/models/translation';
+import IntlService from '../app/services/intl';
+
+interface Hooks {
+  beforeEach: (fn: Function) => any;
+}
```

Comment:
I think `ember-qunit` already exposes `NestedHooks`.

Reply:
`ember-qunit` defines `NestedHooks`, but from what I could tell it isn't exported